BitwardenShared/Core/Auth/Repositories/AuthRepository.swift

```diff
@@ -1031,4 +1019,13 @@ extension DefaultAuthRepository: AuthRepository {
         if let maybeId { return maybeId }
         return try await stateService.getActiveAccountId()
     }
+
+    private func configureBiometricUnlockIfRequired() async throws {
+        if case .available(_, true, false) = try? await biometricsRepository.getBiometricUnlockStatus() {
+            try await biometricsRepository.configureBiometricIntegrity()
+            try await biometricsRepository.setBiometricUnlockKey(
+                authKey: clientService.crypto().getUserEncryptionKey()
+            )
+        }
+    }
```

Comment:
⛏️ Add docs.
❓ Would it be ok the unlock flows affected by this break if this throws? I'd guess it's fine but checking just in case.

Reply:
yeah I was thinking that if this fails for some reason then the calls wouldn't be executed and the user would have to provide another form of auth.  Instead of being given the optional to provide biometrics. Does that make sense?